code/modules/clothing/under/syndicate_jumpsuits.dm

```diff
@@ -21,6 +21,14 @@
 	icon_state = "syndicate_combat"
 	item_color = "syndicate_combat"
 
+
+/obj/item/clothing/under/syndicate/greyman
+	name = "greyman henley"
+	desc = "Attire for someone who finds it hard to survive in the safest place known to man - urban environment "
+	icon_state = "greyman"
+	item_color = "greyman"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 5, FIRE = 40, ACID = 35)
```

Comment:
```suggestion
	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 0, FIRE = 50, ACID = 35)
```

Armor should be identical to the tacticool variant.

Reply:
I wanted to give it a lil insignificant fluff difference

Reply:
Fair enough. resolving.